snuba/query/processors/uuid_column_processor.py

```diff
@@ -0,0 +1,152 @@
+from typing import List, Set
+import uuid
+
+from snuba.query.conditions import (
+    binary_condition,
+    ConditionFunctions,
+    FUNCTION_TO_OPERATOR,
+)
+from snuba.query.expressions import Column, Expression, FunctionCall, Literal
+from snuba.query.logical import Query
+from snuba.query.matchers import (
+    AnyOptionalString,
+    Or,
+    Param,
+    String,
+)
+from snuba.query.matchers import Column as ColumnMatch
+from snuba.query.matchers import FunctionCall as FunctionCallMatch
+from snuba.query.matchers import Literal as LiteralMatch
+from snuba.query.processors import QueryProcessor
```

Comment:
I think this needs to be the Clickhouse QueryProcessor, not the logical query one.